firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -72,6 +78,14 @@
   private final Object lockGenerateFid = new Object();
   /* file used for process-level syncronization of generating and persisting fids */
   private static final String LOCKFILE_NAME_GENERATE_FID = "generatefid.lock";
+  /* File used to read the version of Instance-ID(IID) SDK if it exists in the dependency tree.
+  This is needed to prevent incompatible versions of IID SDK to be used in parallel with FIS SDK. */
+  private static final String IID_SDK_PROP_FILE = "/firebase-iid.properties";
+  /* List of all Instance-ID versions that are incompatible with FIS SDK.*/
+  private static final List<String> IID_SDK_VERSIONS_INCOMPATIBLE_WITH_FIS =
+      Arrays.asList("20.0.2", "20.0.1", "20.0.0", "19.0.1", "19.0.0", "18.0.0");
```

Comment:
What if I use an even older version? like 17.1.2?

Reply:
Changed it to check for any older version than 20.1.1 ( IID with FIS)